linkerd/docs/client_tls.md

```diff
@@ -87,3 +87,44 @@ routers:
           certPath: /certificates/cert.pem
           keyPath: /certificates/key.pem
 ```
+
+### Client TLS and transformers
+
+When using any kind of transformer with client TLS using `io.l5d.static`, ensure that the client
```

Comment:
I think this can benefit from a bit more background:

> If you use transformers, each transformer will prepend a transformer prefix to the client name to indicate that it has been transformed.  Transformer prefixes always start with `/%`.  When using the `io.l5d.static` client configuration, ensure that the client prefix includes the transformer prefix.  For example....